lib/src/components/NxButton/__tests__/NxButton.test.tsx

```diff
@@ -65,4 +65,37 @@ describe('NxButton', function() {
     expect(() => render(<NxButton variant="icon-only" title="Save" disabled />))
         .toThrow(TypeError);
   });
+
+  it('applies small modifier class when provided in className', function() {
+    const smallBtn = quickRender({ className: 'nx-btn--small' }).getByRole('button');
+    expect(smallBtn).toHaveClass('nx-btn--small');
+  });
+
+  it('works with small modifier across different button variants', function() {
```

Comment:
IMO this test isn't necessary either.  While it's true that we don't have separate versions of the "it adds specified classNames to the element in addition to the defaults" for each `variant`, I think it would be excessive to do so.  Also the classnames for the variants (such as `.nx-btn--secondary`) are implementation details and should not be asserted in unit tests.  Only the stylistic results of those classes matter externally, and those are checked in the visual tests.

Reply:
removed